apps/examples/wc-nuxt2/pages/index.vue

```diff
@@ -1,42 +1,42 @@
-<template>
```

Comment:
no changes in this file 👍 I used it to test but not sure why it's removed and re-added the code